app/controllers/index.js

```diff
@@ -69,16 +57,17 @@ class DefaultController extends EventEmitter {
       this.emit('get', req[this.modelName].toObject());
       res.json(req[this.modelName]);
     } else {
-      const errorMsg = `Cannot get model, request does not have a value linked to key: ${this.modelName}`;
+      const errorMsg = `get failed: Cannot get model, request does not have a value linked to key: ${this.modelName}`;
       winston.warn(errorMsg);
-      res.status(300).json({ error: errorMsg });
+      res.status(500).json({ error: errorMsg });
     }
   }
 
   find(req, res) {
-    this._model.query(req.query, (error, list) => {
+    this._model.find(req.query, (error, list) => {
```

Comment:
why this change? all models are extended with `mongoose-query` -plugin (like [this](https://github.com/OpenTMI/opentmi/blob/master/app/models/resource.js#L178)) so there should be also `model.query()`-api available. This is important because find doesn't support features we need from TMI API..